src/test/java/net/minecraftforge/debug/ClientExceptionTestMod.java

```diff
@@ -0,0 +1,61 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiErrorScreen;
+import net.minecraftforge.fml.client.CustomModLoadingErrorDisplayException;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+@Mod(modid = "clientexceptiontest", version = "0", name = "Client Exception Test")
```

Comment:
Versioning only matters that we can determine A's relation to B.
The number of segments in your string doesn't matter. All that matters is that its numbers separated by periods.
1 == 1.0.0.0.0.0.0....
1 > 0.2 as it will be treated at 1.0 vs 0.2